account_financial_report_qweb/models/account_move_line.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-
+from odoo import api, models
+
+
+class AccountAccount(models.Model):
+    _inherit = 'account.move.line'
+
+    @api.model_cr
+    def init(self):
```

Comment:
I think you should indicate in a docstring why we are adding this for our future we to know what we are doing.

Reply:
Done